src/goto-programs/goto_contractor.cpp

```diff
@@ -17,8 +17,16 @@ void goto_contractort::get_constraints(goto_functionst goto_functions)
 {
   auto function = goto_functions.function_map.find("c:@F@main");
   for(const auto &ins : function->second.body.instructions)
+  {
     if(ins.is_assert())
       constraint = create_constraint_from_expr2t(ins.guard);
+    else if(ins.is_function_call())
```

Comment:
this is an `if` followed by another `if`. Can you inline it? For example, instead of `if(cond1) if(cond2)` you can do `if(cond1 && cond2)`

Reply:
done